docs/about/release_notes.rst

```diff
@@ -27,6 +27,103 @@ Release Notes
  Miscellaneous:
    -
 
+Version 4.1
+------------
+
+Breaking changes:
+  - Disable automatic resize of state-map when setting outside map size (`#3951 <https://github.com/equinor/ert/pull/3951>`_)
+  - Remove the GEN_PARAM keyword (`#3867 <https://github.com/equinor/ert/pull/3867>`_)
+  - Move init and storing of GEN_KW form C to python (`#3943 <https://github.com/equinor/ert/pull/3943>`_)
+  - Simplify EclConfig (`#3919 <https://github.com/equinor/ert/pull/3919>`_)
+  - Change default runpath (`#4019 <https://github.com/equinor/ert/pull/4019>`_)
+  - Remove no longer used min std (`#4057 <https://github.com/equinor/ert/pull/4057>`_)
+  - Remove RSH queue driver (`#3962 <https://github.com/equinor/ert/pull/3962>`_)
+  - Replace libecl RNG with numpy.random.Generator (`#4033 <https://github.com/equinor/ert/pull/4033>`_)
+
+Bugfixes:
+  - Make default ensemble path consistent (`#3982 <https://github.com/equinor/ert/pull/3982>`_)
+  - Update torque driver to handle finished jobs (`#3880 <https://github.com/equinor/ert/pull/3880>`_)
+  - Fix bug where extra case is created when running ies (`#4049 <https://github.com/equinor/ert/pull/4049>`_)
+  - Make sure loading gui produces just one storage folder (`#4053 <https://github.com/equinor/ert/pull/4053>`_)
+  - Add grid and grid_file properties back to libres_facade (`#4063 <https://github.com/equinor/ert/pull/4063>`_)
+  - Disable "Start simulation" button while running simulations (`#4080 <https://github.com/equinor/ert/pull/4080>`_)
+  - Show restart button when all realisations fail (`#4113 <https://github.com/equinor/ert/pull/4113>`_)
+  - Propagate error messages from job_queue_node (`#4127 <https://github.com/equinor/ert/pull/4127>`_)
+  - Propagate qstat options to qstat executable (`#4138 <https://github.com/equinor/ert/pull/4138>`_)
```

Comment:
This can be muted, it is not a bugfix relative to previous release, only to a previous commit.